api/src/controllers/changes.js

```diff
@@ -189,13 +191,15 @@ const getChanges = feed => {
     }
   }
 
-  feed.changesReqs = [];
+  const upstreamSession = { id:upstreamSessionId, requests:[] };
```

Comment:
The word "session" confuses me here because it makes me think of http sessions. Maybe call it `upstreamRequest`? This also applies everywhere else "session" is used.